### DISCUSSION TITLE
Can you merge a folder of video clips (like from the same gopro camera session)?

### DISCUSSION THREAD

**Discussion by gth001** [mifi/lossless-cut#1845] [Support] [answered]
I currently do so (without any re-encoding) with this:

`find *.MP4 | sed 's:\ :\\\ :g'| sed 's/^/file /' > fl.txt; ffmpeg -f concat -i fl.txt  -c:a aac -c:v copy output.mp4; rm fl.txt`

but I was wondering if lossless cut also had this feature? If not it's ok, I can stick to my terminal code. I just figured it would fit in with the functionality well.

**Comment by mifi** [comment, marked as answer]
yes, losslesscut can do this. drag drop all the files into losslesscut then click the merge/concat button in the batch file list that pops up

**Reply by gth001** [reply]
Awesome, that worked perfectly. Love this app.
